packages/common/src/hooks/useGatedContent.ts

```diff
@@ -54,8 +58,8 @@ export const useGatedContentAccess = (track: Nullable<Partial<Track>>) => {
 
       return {
         isFetchingNFTAccess: !hasNftAccessSignature && isSignatureToBeFetched,
-        hasStreamAccess: !!stream,
-        hasDownloadAccess: !!download
+        hasStreamAccess: !isStreamGated || !!stream,
```

Comment:
will this break if either of the fields don't exist?
eg. if `isStreamGated` is `undefined` will `hasStreamAccess = true`?

Reply:
isStreamGated will always exist